### PR TITLE
Token scanner updates

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

File: src/pages/docs/api-security/token-scanner.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ search_keyword: "password security, secret scanning, secret, API key security"
 ---
 The Postman Token Scanner scans your public workspaces, collections, environments, and documentation to find accidentally exposed tokens. This protects your organization and prevents malicious users from exploiting the tokens.
 
-> Token Scanner is available only on the [Enterprise plan](https://www.postman.com/pricing/) and is enabled by default.
+> Token Scanner is available on all Postman plans and is enabled by default. The custom alert is available only on [Enterprise plan](https://www.postman.com/pricing/).
```
Comment (tirth055) [inline src/pages/docs/api-security/token-scanner.md:9]: Wouldn't it be better if we add this blockquote for custom alerts inside the custom alerts section on this page?

**Review by tirth055 — APPROVED** [review]

File: src/pages/docs/api-security/token-scanner.md [hunk 2]
```diff
@@ -57,10 +57,12 @@ By default, tokens issued by the following service providers are scanned:
 
 Custom alerts can be used to scan your team's proprietary and third-party app tokens that are not scanned by default.
 
-> Your team can add a total of five alerts.
+> **[Custom alert is available only on Postman Enterprise plan](https://www.postman.com/pricing/)**.
+
+Your team can add a total of five alerts. You should be a **Community Manager** or member with both **Developer** and **Admin** roles to add custom alerts.
```
Comment (jkonrath-postman) [inline src/pages/docs/api-security/token-scanner.md:62]: Change should to must

File: src/pages/docs/api-security/token-scanner.md [hunk 3]
```diff
@@ -57,10 +57,12 @@ By default, tokens issued by the following service providers are scanned:
 
 Custom alerts can be used to scan your team's proprietary and third-party app tokens that are not scanned by default.
 
-> Your team can add a total of five alerts.
+> **[Custom alert is available only on Postman Enterprise plan](https://www.postman.com/pricing/)**.
```
Comment (jkonrath-postman) [inline src/pages/docs/api-security/token-scanner.md:60]: "Custom alerts are" instead of "Custom alert is" ?

**Review by jkonrath-postman — APPROVED** [review]
Two minor things, but otherwise approved

**Pull request merged**